### PR TITLE
Add parsing for blend mode (bm) on ShapeLayerContent.

### PR DESCRIPTION
**Pull request by simeoncran**

### CODE REVIEW

File: source/LottieData/Polystar.cs [hunk 1]
```diff
@@ -21,7 +20,7 @@ sealed class Polystar : ShapeLayerContent
             Animatable<double> outerRadius,
             Animatable<double> innerRoundedness,
             Animatable<double> outerRoundedness)
-            : base(name, matchName)
+            : base(args)
```
Comment (plaiMS) [inline source/LottieData/Polystar.cs]: >args) [](start = 19, length = 5)

in args

Comment (simeoncran) [reply source/LottieData/Polystar.cs]: Already fixed in the latest commit... must have been slow to arrive.

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

not used anymore

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1069 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

not used anymore

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1099 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

remove

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1136 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

remove

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1156 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

remove

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1169 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

remove

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1246 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

remove

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1265 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

remove

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1278 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

File: source/LottieReader/Serialization/LottieCompositionReader.cs [hunk 2]
```diff
@@ -1278,24 +1282,23 @@ TrimPath ReadTrimPath(JObject obj)
...
                 trimType,
                 startPercent,
                 endPercent,
                 offsetDegrees);
         }
 
-        Repeater ReadRepeater(JObject obj)
+        Repeater ReadRepeater(JObject obj, in ShapeLayerContent.ShapeLayerContentArgs shapeLayerContentArgs)
         {
             var name = ReadName(obj);
```
Comment (plaiMS) [inline source/LottieReader/Serialization/LottieCompositionReader.cs]: >var name = ReadName(obj); [](start = 12, length = 25)

remove

**Comment by plaiMS** [comment]
>             var name = ReadName(obj);

remove

---
Refers to: source/LottieReader/Serialization/LottieCompositionReader.cs:1306 in c2f8fd6. [](commit_id = c2f8fd66a8cf030c17522a9614967d516d586347, deletion_comment = False)

File: source/LottieReader/Serialization/LottieCompositionReader.cs [hunk 3]
```diff
@@ -1319,8 +1321,7 @@ RoundedCorner ReadRoundedCorner(JObject obj)
             var radius = ReadAnimatableFloat(obj.GetNamedObject("r"));
```
Comment (plaiMS) [inline source/LottieReader/Serialization/LottieCompositionReader.cs]: remove

**Pull request merged**